Library/Homebrew/os/linux/global.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 # enables experimental readelf.rb, patchelf support.
```

Comment:
Not experimental any more. Let's just remove all usage of `HOMEBREW_PATCHELF_RB` and `HOMEBREW_NO_PATCHELF_RB` and delete the legacy code. We can revert this PR if we need to turn it off after that.

Reply:
Agreed with this; could you make these changes @rmNULL?